package.json

```diff
@@ -51,7 +51,7 @@
     "rxjs": "^5.5.0",
     "semaphore-async-await": "^1.5.1",
     "string-similarity": "^1.1.0",
-    "typescript": "2.4.2",
+    "typescript": "~2.6.2",
```

Comment:
I don't know if we can guarantee that TS will still work on patch version upgrades. We also don't currently have a lockfile so this would be very risky.

Reply:
No problem!  I'll remove the tilde.

Reply:
Done!